addons/web/static/src/scss/bootstrap_review_backend.scss

```diff
@@ -253,3 +253,23 @@ $-o-bg-colors-custom: null;
     background-color: $input-focus-bg;
     border-color: $o-input-hover-border-color;
 }
+
+// Switches
+//    Creating a different styling for toggle switches with a purpose of switching
```

Comment:
The description is good, bit it should be placed just before the `.o_switch_toggle` selector 😉
You can remove the indentation (I think that you wanted to match the prev. comments, but those were list points).
In conclusion something like (note classes wrapped in '' ):
```
// Switches
// ============================================================================

// Creating a different styling for toggle switches with a purpose of switching
// from A to B (eg. switching between month and year) rather than ON/OFF.
// The 'o_switch_toggle' class should be use in addition to the default
// 'form-switch' Bootstrap class.
.o_switch_toggle { ... }
```